pages/2015/index.rst

```diff
@@ -86,7 +86,7 @@ conduct for the workshop which can be found
 Community Code of
 Conduct <https://www.python.org/psf/codeofconduct/>`__. If you have any
 questions about the workshop, you can reach the SOC at
-python-in-astronomy-soc@googlegroups.com
+python.in.astronomy.soc@googlegroups.com
```

Comment:
Ah also change googlegroups.com to gmail.com - the old address was a group we never ended up using that much